src/modules/io.c

```diff
@@ -310,3 +310,32 @@ FILESYSTEM_createDirectory(WrenVM *vm) {
     VM_ABORT(vm, "Directory could not be created");
   }
 }
+
+internal void
+FILESYSTEM_doesFileExist(WrenVM* vm)
+{
+ ASSERT_SLOT_TYPE(vm, 1, STRING, "file path");
+ const char* path = wrenGetSlotString(vm, 1);
+ ENGINE* engine = (ENGINE*)wrenGetUserData(vm);
+ 
+ bool fileCheck = ENGINE_fileExists(engine, path);
+ 
+ if (!fileCheck) {
+   wrenSetSlotBool(vm, 0, false);
+   return;
+ }
+ wrenSetSlotBool(vm, 0, true);
```

Comment:
We can replace all of this with:
```
wrenSetSlotBool(vm, 0, !fileCheck);
```

Reply:
Fixed this too changed to the smaller code